### PR TITLE
docs: add sample application for testing purposes

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR adds an example project in the form of a game server processor using MQTT and MongoDB.

### CODE REVIEW

**Comment by jonaslagoni** [comment]
@NektariosFifes this is the example project. I think it makes sense it is added directly to the repo for easy access for all contributors. Thoughts?

File: example-projects/game-processor/asyncapi.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    url: "0.0.0.0:1883"
+    description: "Development server",
+    protocol: "mqtt"
+defaultContentType: application/json
+channels: 
+  game/server/{serverId}/events/player/{playerId}/item/{itemId}/pickup:
+    description: Channel used when a player picks up an item in-game
+    parameters:
+      serverId: 
+        $ref: "./components/Parameters.yaml#/serverId"
```
Comment (NektariosFifes) [inline example-projects/game-processor/asyncapi.yaml:19]: why not ./AsyncAPI/components/Parameters.yaml#/serverId

Comment (jonaslagoni) [reply example-projects/game-processor/asyncapi.yaml:19]: Wups! Copy paste mistake, gonna fix it tomorrow 🙂

Comment (jonaslagoni) [reply example-projects/game-processor/asyncapi.yaml]: Fixed 👍

**Review by NektariosFifes — APPROVED** [review]
seems ok to me :)

**Review by fmvilas — APPROVED** [review]

**Pull request merged**